package.json

```diff
@@ -47,10 +47,11 @@
     "shelljs": "^0.8.5"
   },
   "overrides": {
-    "autolinker": "^4.0.0"
+    "autolinker": "^4.0.0",
+    "vue-template-compiler": "@vue/compiler-sfc@3.4.31"
```

Comment:
Is this a temporal way?
Do you follow up when we can remove this modules?

Reply:
Both are temporal ways.
`autolinker` is a submodule of `markdown-toc`, `vue-template-compiler` is a submodule of `documentation`, and to restore that code, the 3rd party module must delete those modules.

Note that we cannot add comments in package.json, so we need to access the PR through git blame and check the history.